README.md

```diff
@@ -55,6 +61,114 @@ You can start editing the page by modifying `pages/index.js`. The page auto-upda
 
 The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
 
+## Adding remote repositories
+
+Configure the registry viewer through environment variables.
+
+- `DEVFILE_SOURCE_REPOS` - Source repository(ies) name
+- `DEVFILE_ENDPOINTS` - Endpoint(s) that serves the devfile JSON
+- `DEVFILE_DIRECTORIES` - Endpoint(s) that serves the devfile
+
+Notes about environment variables:
+
+- The environment variables DEVFILE_SOURCE_REPOS, DEVFILE_ENDPOINTS, and DEVFILE_DIRECTORIES must be the same array length and in order.
```

Comment:
It might be helpful to provide an example of configuring with multiple source repos via env vars?

Reply:
Another possible thought is to have one ENV rather than 3 and having a defined format to mention the key, index endpoint and devfile endpoint

Reply:
Fixed